### ISSUE TITLE
Redshift uses the deleted file /memfd:pulseaudio

### ISSUE THREAD

**Issue by mYnDstrEAm** [jonls/redshift#789]
Happy new year. 

**Describe the bug**
Running `sudo rkhunter -c --enable all --disable none --rwo` tells me that redshift is using the deleted file /memfd:pulseaudio. Why is that? It seems to be a bug and should probably be prevented. I don't think redshift needs / should access/use anything related to audio/pulseaudio.

- [x] [I have checked the FAQ](https://github.com/jonls/redshift/blob/master/README.md#faq) and my issue is not mention there.

**To Reproduce**
1. Install/use Debian10/KDE
2. Install/use redshift widget
3. Run the command above to see that it accesses this deleted file

**Expected behavior**
It should not use deleted files. Furthermore, it should also not use anything related to audio.

**Error output/logs/screenshots**
I also asked about it [here](https://unix.stackexchange.com/questions/624856/why-does-redshift-use-the-deleted-file-memfdpulseaudio).

**Software versions (please complete the following information):**
 - OS: Linux/Debian10 stable
 - Redshift version: 1.12
 - Distribution: Debian 10 stable
 - Redshift installed from: apt-get

**Comment by CameronNemo** [comment]
/memfd deleted files are pretty typical to expect. rkhunter should be ignoring those, and likely many of these: https://github.com/leahneukirchen/xtools/blob/master/xcheckrestart#L9-L18

**Comment by mYnDstrEAm** [comment]
No, this shouldn't be expected and rkhunter shouldn't be ignoring those. Don't know why it should likely be ignoring many of the ones linked there - you didn't explain that at all. I do know that dimming the screen / changing the screen-color shouldn't need anything related to audio - whether deleted or not.

**Comment by CameronNemo** [comment]
whatever. /memfd files are not real files and they are completely normal. they never exist. glib might be doing something weird (redshift links to it usually).
